src/logic/hooks/useEstimateTransactionGas.tsx

```diff
@@ -236,7 +237,12 @@ export const useEstimateTransactionGas = ({
         const estimatedGasCosts = gasEstimation * parseInt(gasPrice, 10)
         const gasCost = fromTokenUnit(estimatedGasCosts, nativeCoin.decimals)
         const gasCostFormatted = formatAmount(gasCost)
-        const gasLimit = (gasEstimation * 2).toString()
+        const gasLimit = (
+          gasEstimation * 2 +
+          MINIMUM_TRANSACTION_GAS +
```

Comment:
@dasanra  one question, `gasEstimation` already add  MINIMUM_TRANSACTION_GAS + SIGNATURE_COST +  SAFE_TX_GAS_DATA_COST in src/logic/safe/transactions/gas.ts:229, why is also needed to be added here?